### DISCUSSION TITLE
School Issued Faculty Email Address Not Recognized

### DISCUSSION THREAD

**Discussion by emreamasyali** [community/community#148006] [GitHub Education] [answered]
### Hi everyone,

I am a postdoc at a university (and a course lecturer) and am trying to register for GitHub education for teachers. My school appears on the dropdown list for the application; however, while the @student emails are recognized, my faculty email associated with my GitHub account is not accepted. Does anyone else have the same issue?

**Comment by davevad93** [comment, marked as answer]
Hi @emreamasyali ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "` My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.

**Reply by emreamasyali** [reply]
Thank you; I just created a ticket
